SU2_CFD/include/fluid/CFluidScalar.hpp

```diff
@@ -43,13 +43,58 @@ class CFluidScalar final : public CFluidModel {
   su2double Gamma;                  /*!< \brief Ratio of specific heats of the gas. */
   su2double Pressure_Thermodynamic; /*!< \brief Constant pressure thermodynamic. */
 
-  std::vector<su2double> molarMasses;  /*!< \brief Molar masses of all species. */
+  bool wilke;
+  bool davidson;
+
+  std::vector<su2double> massFractions;              /*!< \brief Mass fractions of all species. */
+  std::vector<su2double> moleFractions;              /*!< \brief Mole fractions of all species. */
+  std::vector<su2double> molarMasses;                /*!< \brief Molar masses of all species. */
+  std::vector<su2double> laminarViscosity;           /*!< \brief Laminar viscosity of all species. */
+  std::vector<su2double> laminarThermalConductivity; /*!< \brief Laminar thermal conductivity of all species. */
+
+  static const int ARRAYSIZE = 100;
+  std::unique_ptr<CViscosityModel> LaminarViscosityPointers[ARRAYSIZE];
+  std::unique_ptr<CConductivityModel> ThermalConductivityPointers[ARRAYSIZE];
+
+  /*!
+   * \brief Convert mass fractions to mole fractions.
+   * \param[in] val_scalars - Scalar mass fraction.
+   */
+  std::vector<su2double>& massToMoleFractions(const su2double* val_scalars);
+
+  /*!
+   * \brief Wilke mixing law for mixture viscosity.
+   * \param[in] val_scalars - Scalar mass fraction.
+   */
+  su2double wilkeViscosity(const su2double* val_scalars);
+
+  /*!
+   * \brief Davidson mixing law for mixture viscosity.
+   * \param[in] val_scalars - Scalar mass fraction.
+   */
+  su2double davidsonViscosity(const su2double* val_scalars);
```

Comment:
Please be consistent... all other function of this class are CamelCase, do the same here please

Reply:
Ok, Thank you so much.